### PR TITLE
feat(styles): implement form switch styles using tokens

### PR DESCRIPTION
**Pull request by schaertim**

### CODE REVIEW

**Review by alizedebray — CHANGES_REQUESTED** [review]
Here are the things I noticed: 
- [x] The active switch has no check icon on hover
- [x] The focus outline is 100% wide where it should hug the content
- [x] The focus outline should be rounder when the switch is used with a hidden label
- [x] The disabled styles are not correct 
- [x] The invalid styles seems incorrect as well but you can first ask Sandra to document them in Figma
- [x] I would probably set the label after by default in the docs since it seems to be the default in Figma
- [x] You will have to update [the snapshot story](https://preview-3558--swisspost-design-system-next.netlify.app/iframe.html?id=snapshots--switch&viewMode=story) to use the correct tokens on dark background
- [x] The title of the PR is incorrect it should be like so: "feat(package): summary of your changes"

Do not hesitate to contact me if something is unclear.

File: packages/styles/src/components/form-check.scss
[the remark this answers is not in the document]
Comment (schaertim) [reply packages/styles/src/components/form-check.scss:1]: Here is the new ticket about the styles separation: #3628

**Comment by schaertim** [comment]
I implemented the PR feedback, except for these two points:

-  The invalid styles seems incorrect as well but you can first ask Sandra to document them in Figma
- You will have to update [the snapshot story](https://preview-3558--swisspost-design-system-next.netlify.app/iframe.html?id=snapshots--switch&viewMode=story) to use the correct tokens on dark background

Regarding the first one: The validation styles will not be implemented for each component using individual tokens respective to each component. Instead the styling will be done centrally with a designated "validation" token set to be reused in all components requiring validation. I opened a new ticket (#3630) for this which has to be tackled first before completing the switch styling.

And for the second point: Right now it is not clear if the inverted mode will differ from the dark mode. If it does new tokens would have to be created for all inverted colors. Discussions about that decision are still ongoing.

File: packages/styles/src/components/form-check.scss [hunk 1]
```diff
@@ -206,61 +210,162 @@
...
+      }
     }
 
     &[disabled] {
-      background: form-check.$form-switch-disabled-bg;
+      background-color: tokens.get('switch', 'element-disabled-bg');
+      border: tokens.get('switch', 'border-width') tokens.get('switch', 'border-style-disabled')
+        tokens.get('switch', 'element-disabled-stroke');
+
+      @include color-mx.on-dark-background {
```
Comment (alizedebray) [inline packages/styles/src/components/form-check.scss:336]: Light and dark modes are now handled by the tokens through the semantic layer for modes, you shouldn't have to do anything manually.

**Review by alizedebray — CHANGES_REQUESTED** [review]

**Review by alizedebray — CHANGES_REQUESTED** [review]
Can you fix the e2e tests please?

**Pull request closed without merging**